src/super_gradients/recipes/dataset_params/pascal_voc_detection_dataset_params.yaml

```diff
@@ -0,0 +1,61 @@
+train_dataset_params:
+  data_dir: ./data/pascal_voc/
+  input_dim: [320, 320]
+  cache: False
+  cache_dir:
+  transforms:
+  - DetectionPaddedRescale:
+      input_dim: ${dataset_params.train_dataset_params.input_dim}
+  - DetectionTargetsFormatTransform:
+      max_targets: 50
+      output_format:
+        _target_: super_gradients.training.utils.detection_utils.DetectionTargetsFormat # targets format
```

Comment:
don't we have a factory for this?

Reply:
No, notice we resave here args for transforms.

I think the topic of resolving args for classes resolved by factories is a different topic, a one we should discuss how to do outside this PR as it might lead us into trouble when done wrong.